num_enum/tests/renamed_num_enum.rs

```diff
@@ -1,13 +1,36 @@
+#![cfg_attr(rustfmt, rustfmt::skip)] // args
```

Comment:
Why skip the formatting? While I don't love what rustfmt does with this file, I think I still prefer consistency...

Reply:
Fair enough, will strip that